Demo/Demo/ViewModels/BaseViewModel.swift

```diff
@@ -216,6 +238,29 @@ class BaseViewModel: ObservableObject, PayPalWebCheckoutDelegate, CardDelegate {
         updateTitle("3DS challenge has finished")
         print("3DS challenge has finished")
     }
+    
+    // MARK: - CardVault Delegate
+    
+    func card(_ cardClient: CardClient, didFinishWithVaultResult vaultResult: CardVaultResult) {
+        updateTitle(
+            "Vault without Purchase has finished. \n SetupTokenID: \(vaultResult.setupTokenID) \nVault Status: \(vaultResult.status)"
+        )
+        print("Vault without purchase has finished: \(vaultResult)")
+        Task {
+            if let paymentTokenResult = try? await DemoMerchantAPI.sharedService.getPaymentToken(
+                setupToken: vaultResult.setupTokenID, selectedMerchantIntegration: selectedMerchantIntegration
+            ) {
+                print("Payment Token: \(String(describing: paymentTokenResult))")
```

Comment:
The final payment token is for them to generate on merchant's server side. The output is just for our test purposes.

Reply:
I am working on separate CardVaultViewModel and a separate CardVaultView to handle the flow separately. For this PR, I am using the existing model of settings menu to toggle options, "CardVault" being one. I started on another branch deleting all UIKit components and the third party library for managing settings menu data in favor of reactive SwiftUI views and SwiftUI navigation.

Reply:
> The final payment token is for them to generate on merchant's server side.
So for the purposes of our SDK, we are done at getting vaultResult. the Task block is the
part that would be moved to pressing button next for the merchant to create a final Payment Token on their server side and that is just for the demo app to test the flow.

Since the client-side flow will never involve a paymentToken, I don't think we need to send it back to the demo app at all.

Reply:
I just tacked it on the delegate function so we can see the final flow, but since it is handled in the [PR for UI for vault without purchase](https://github.com/paypal/paypal-ios/pull/183), I will delete it.

Reply:
addressed ae732e8b2187cf2b561bad7b601c611e573bb23a

Reply:
Whatever we _do_ decide though, the Demos should function the same across iOS & Android. Currently they've diverged.

Reply:
It's handled in this PR. I had originally tacked it onto the delegate function because there wasn't 
"get Payment Token" button after returning from SDK yet. 
But all the UI elements are built here. https://github.com/paypal/paypal-ios/pull/183